### PR TITLE
Create psammead-most-read component and add MostReadTitle

### LINKED ISSUE #580: Create psammead-most-popular Title
**Is your feature request related to a problem? Please describe.**
The most read component should have a title that reads 'Most read'. This should be a file within the `@bbc/psammead-most-read` component. It should remain an alpha component.

image
Zeplin designs:  https://app.zeplin.io/project/5baa564c05c2eba0d2b89568/dashboard?tag=Most%20read

Acceptance Criteria
    Given I use a screen reader
    When I navigate to the Most Read section
    Then a heading level 2 must be announced as the first piece of content
    And the h2’s announced text should be the same as the region landmark’s label for this section
    And the h2 must be unique to the page

Note - the landmark label will be created when the wrapper for the most-read component is created.

**Describe the solution you'd like**
Create a styled-component in `psammead-most-popular` that implements this heading.

- An h2 element
- Font Reith Sans Regular for `Latin` and Regular weight for other non-latin scripts. 
- Colour `C_Shadow`
- Type `Trafalgar`

Similar to SubHeading component: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-headings/src/index.jsx#L24-L35

**Describe alternatives you've considered**
Modify `psammead-section-label` to also satisfy these requirements. Or we could re-use the article subheading - however it has padding and spacing and bold weight, so it's better to have a specific heading for most read.

**Testing notes**
[Tester to complete]

Dev insight: Unit snapshot tests and stories for storybook should be sufficient.

**Additional context**

### PR DESCRIPTION
**Pull request by hotinglok**
Resolves #580, ~waiting on #2491 to be merged first.~

~**Changes to be made once #2491 is merged**~
~- Update version.~
~- Re-add in `MostReadItem`~

**Overall change:**
Created `psammead-most-read` component and added `MostReadTitle` associated files.

**Code changes:**
- Created `psammead-most-read` component
- Added `title.jsx`, `title.test.jsx`, `title.stories.jsx`.
- Removed `item.jsx`, `item.test.jsx`, `item.stories.jsx` ~until #2491 is merged.~ due to talks about refactoring so these will be re-added later.
- Updated readme and version.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-most-read/src/title.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string } from 'prop-types';
+import { C_SHADOW } from '@bbc/psammead-styles/colours';
+import { getTrafalgar } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const MostReadTitleWrapper = styled.h2`
+  ${({ script }) => script && getTrafalgar(script)};
```
Comment (sareh) [inline packages/components/psammead-most-read/src/title.jsx:10]: The styling of types and colours looks good. 👍

File: packages/components/psammead-most-read/src/title.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styled from 'styled-components';
+import { shape, string } from 'prop-types';
+import { C_SHADOW } from '@bbc/psammead-styles/colours';
+import { getTrafalgar } from '@bbc/gel-foundations/typography';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+
+const MostReadTitleWrapper = styled.h2`
```
Comment (sareh) [inline packages/components/psammead-most-read/src/title.jsx]: It'd be better to name styled components related to what the underlying HTML element is. e.g.
```suggestion
const StyledHeading = styled.h2`
```
This is in keeping with the naming convention in other Psammead packages.

**Review by thekp — COMMENTED** [review]
Can you update the readme to include prop requirements for most read title, similar to this readme: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-story-promo/README.md

**Edit:**

This title is probably worth adding as well: 
## ⛔️ This is an alpha component ⛔️

File: packages/components/psammead-most-read/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+const item = {
+    header: 'Cranberries singer O'Riordan died by drowning,
+    href: 'https://www.bbc.com'
+}
+
+<MostRead item={item} count="1" service="news" dir="ltr" />;
+
+```
+
+![most-read-single-item](documentation/most-read-single-item.png)
```
Comment (thekp) [inline packages/components/psammead-most-read/README.md]: I don't think we need to have a static image in our package, we can just use github's hosting:
image

`image`

File: packages/components/psammead-most-read/README.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    script={latin} 
+    service="news" 
+    dir="ltr"
+/>;
+
+```
+
+Screenshot
+
+![most-read-item-with-last-Updated-date](documentation/most-read-item-with-last-Updated-date.png)
```
Comment (thekp) [inline packages/components/psammead-most-read/README.md]: this should be removed

edit: or replaced with a working image

**Review by thekp — APPROVED** [review]
LGTM, thanks for updating the readme 👍

File: packages/components/psammead-most-read/README.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+## Description
+
+The `MostReadItem` component is designed to display the most read articles given a designated period of time which is dependent on service. The component comprises of a `MostReadTitle`, a `MostReadList` which is a grid containing a `MostReadCount/Rank` (name subject to change) and a `MostReadItem`.
```
Comment (sareh) [inline packages/components/psammead-most-read/README.md]:
```suggestion
The `MostRead` component is designed to display the most read articles given a designated period of time which is dependent on service. The component comprises of a `MostReadTitle`, a `MostReadList` which is a grid containing a `MostReadCount/Rank` (name subject to change) and a `MostReadItem`.
```

**Review by sareh — APPROVED** [review]
👍 Looks great.

**Comment by sareh** [comment]
Since this is an alpha component, we are going to merge this in without the full browser testing. We will carry that out when we're ready to move `most-read` out of alpha.

**Pull request merged**